TruePath/AbsoluteDirectory.cs

```diff
@@ -0,0 +1,25 @@
+// SPDX-FileCopyrightText: 2024 TruePath contributors <https://github.com/ForNeVeR/TruePath>
+//
+// SPDX-License-Identifier: MIT
+
+namespace TruePath;
+
+public static class AbsoluteDirectory
+{
+    public static void CreateDirectory(AbsolutePath path) => Directory.CreateDirectory(path.Value);
+    public static void Delete(AbsolutePath path) => Directory.Delete(path.Value);
+    public static void Delete(AbsolutePath path, bool recursive) => Directory.Delete(path.Value, recursive);
+    public static AbsolutePath GetCurrentDirectory() => new(Directory.GetCurrentDirectory(), false);
+    public static void SetCurrentDirectory(AbsolutePath path) => Directory.SetCurrentDirectory(path.Value);
```

Comment:
Let's implement a setter on `AbsolutePath.CurrentWorkingDirectory` instead (currently it lacks one).